terraform/040-id-broker/vars.tf

```
@@ -43,30 +43,30 @@ variable "contingent_user_duration" {
 }
 
 variable "cpu" {
-  type        = string
+  type        = number
   description = "Amount of CPU to allocate to container, recommend '250' for production"
-  default     = "250"
+  default     = 250
 }
 
 variable "cpu_cron" {
-  type        = string
+  type        = number
   description = "Amount of CPU to allocate to cron container, recommend '128' for production"
-  default     = "128"
+  default     = 128
 }
 
 variable "cpu_email" {
-  type        = string
+  type        = number
   description = "Amount of CPU to allocate to email container"
```

Do we want recommend here?
```suggestion
  description = "Amount of CPU to allocate to email container, recommend '64' for production"
```

---

See above.